CDCgov/prime-reportstream
# 11674: XAD[Address] mapping

This PR adds a completed XAD-Address-XAD mapping along with tests

Test Steps:
1. Confirm that the schema aligns with https://docs.google.com/spreadsheets/d/1hSTEur557TIKPEKZRoprVw-uNpw12JZtri-iQsc4uQ0/edit#gid=0 and that the test cases cover
2. Review: https://github.com/focusconsulting/hl7v2-fhir-converter/pull/6

## Changes
- Implements an Address/XAD.yml schema
- Implements an XAD/Address.y schema

## Checklist

### Testing

### Process

## Linked Issues
- #11674 

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
